src/test/scala/mockws/Helpers.scala

```diff
@@ -0,0 +1,13 @@
+package mockws
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import play.api.mvc.{DefaultActionBuilder, PlayBodyParsers}
+import scala.concurrent.ExecutionContext.Implicits._
+
+object Helpers {
+  implicit val as = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+  val bodyParser = PlayBodyParsers()
+  val action = DefaultActionBuilder(bodyParser.anyContent)
```

Comment:
You are right. Changing the `action` to `Action` is the correct thing to do because of two reasons:
1. It's a member of a Scala _object_ and style dictates it should start with a capital.
2. What you stated above.